src/applications/claims-status/containers/DocumentRedirectPage.jsx

```diff
@@ -0,0 +1,79 @@
+// import { Redirect } from 'react-router-dom';
+import React, { useEffect, useState } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import { Navigate } from 'react-router-dom-v5-compat';
+import { useFeatureToggle } from '~/platform/utilities/feature-toggles';
+import withRouter from '../utils/withRouter';
+import DocumentRequestPage from './DocumentRequestPage';
+
+const DocumentRedirectPage = ({ trackedItem, trackedItemId }) => {
+  const [redirectPath, setRedirectPath] = useState(null);
+  const { TOGGLE_NAMES, useToggleValue } = useFeatureToggle();
```

Comment:
@1Copenut I think you and I were wondering if there was a way to do this last week (?? maybe longer ago, time is crazy) - here is a great example usage! 😅